src/components/StepDefinition/StepDefinition.js

```diff
@@ -46,18 +47,42 @@ const resourceTable = (title, namespace, resources) => {
           )
       }))}
       headers={[
-        { key: 'name', header: 'Name' },
-        { key: 'value', header: 'Value' }
+        {
+          key: 'name',
+          header: intl.formatMessage({
+            id: 'dashboard.taskRun.detailsName',
```

Comment:
could these use `dashboard.tableHeader.name` and `dashboard.tableHeader.value` ?